bundles/org.openhab.binding.vallox/src/main/resources/ESH-INF/thing/se-channels.xml

```diff
@@ -0,0 +1,547 @@
+		<item-type>Number</item-type>
+		<label>Incoming temperature</label>
+		<description>Current temperature of the air flow incoming to the building.</description>
+		<category>Temperature</category>
+		<state min="-20" max="40" step="1" pattern="%d °C" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="seInEfficiency">
+		<item-type>Number</item-type>
+		<label>Incoming efficiency</label>
+		<state min="0" max="100" step="1" pattern="%d %%" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seOutEfficiency">
+		<item-type>Number</item-type>
+		<label>Outgoing efficiency</label>
+		<state min="0" max="100" step="1" pattern="%d %%" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seAverageEfficiency">
+		<item-type>Number</item-type>
+		<label>Average efficiency</label>
+		<state min="0" max="100" step="1" pattern="%d %%" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="sePowerState">
+		<item-type>Switch</item-type>
+		<label>Power State</label>
+		<state readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seCo2AdjustState">
+		<item-type>Switch</item-type>
+		<label>Co2 Adjust State</label>
+		<state readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seHumidityAdjustState">
+		<item-type>Switch</item-type>
+		<label>Humidity Adjust State</label>
+		<state readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="sePostHeatingState">
+		<item-type>Switch</item-type>
+		<label>Post heating State</label>
+		<state readOnly="false"></state>
+	</channel-type>
+
+	<!-- ADVANCED ================================================================ -->
+
+	<channel-type id="seDcFanInputAdjustment" advanced="true">
+		<item-type>Number</item-type>
+		<label>Input fan speed</label>
+		<description>Adjust input fan speed in scale of 0-100%</description>
+		<state min="0" max="100" step="1" pattern="%d %%" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seDcFanOutputAdjustment" advanced="true">
+		<item-type>Number</item-type>
+		<label>Output fan speed</label>
+		<description>Adjust output fan speed in scale of 0-100%</description>
+		<state min="0" max="100" step="1" pattern="%d %%" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seHrcBypassThreshold" advanced="true">
+		<item-type>Number</item-type>
+		<label>Heat recovery cell bypass threshold</label>
+		<description>Heat recovery cell bypass set point temperature</description>
+		<category>Temperature</category>
+		<state min="-20" max="40" step="1" pattern="%d °C" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seInputFanStopThreshold" advanced="true">
+		<item-type>Number</item-type>
+		<label>Input fan Stopping threshold</label>
+		<description>Input fan stops if incoming temperature falls below this.</description>
+		<category>Temperature</category>
+		<state min="-20" max="40" step="1" pattern="%d °C" readOnly="false"></state>
```

Comment:
Please apply openHab's [Units of Measurement](https://www.openhab.org/docs/concepts/units-of-measurement.html) framework wherever applicable. This allows user interfaces greater flexibility when it comes to rendering data from your binding.